doc/source/multiscene.rst

```diff
@@ -231,3 +231,46 @@ multiple Scenes use:
     >>> mscn = MultiScene.from_files(glob('/data/abi/day_1/*C0[12]*.nc'), reader='abi_l1b')
     >>> mscn.load(['C01', 'C02'])
     >>> mscn.save_datasets(base_dir='/path/for/output')
+
+Combining multiple readers
+--------------------------
+
+Since Satpy 0.23, it is possible to automatically
+combine multiple readers into a single MultiScene using the
+:meth:`~satpy.multiscene.MultiScene.from_files` constructor.  It is
+no longer necessary for the user to create the :class:`~satpy.scene.Scene`
+objects themselves.  For example, you can combine Advanced Baseline
+Imager (ABI) and Global Lightning Mapper (GLM) measurements.
+Constructing a multi-reader MultiScene requires more parameters
+than a single-reader MultiScene, because Satpy can poorly guess how
+to group files belonging to different instruments.  For an example
+creating a video with lightning superimposed on ABI channel 14 (11.2
+micrometre), using the built-in composite ``C14_flash_extent_density``,
+which superimposes flash extent density from GLM (read with the
+:class:`~satpy.readers.glm_l2.NCGriddedGLML2` or ``glm_l2`` reader) on ABI
+channel 14 data (read with the :class:`~satpy.readers.abi_l1b.NC_ABI_L1B`
+or ``abi_l1b`` reader), and therefore needs Scene objects that combine
+both readers:
+
+    >>> glm_dir = "/path/to/GLMC/"
+    >>> abi_dir = "/path/to/ABI/"
+    >>> ms = satpy.MultiScene.from_files(
+    ...        glob.glob(glm_dir + "OR_GLM-L2-GLMC-M3_G16_s202010418*.nc") +
+    ...        glob.glob(abi_dir + "C*/OR_ABI-L1b-RadC-M6C*_G16_s202010418*_e*_c*.nc"),
+    ...        reader=["glm_l2", "abi_l1b"],
+    ...        ensure_all_readers=True,
+    ...        group_keys=["start_time"],
+    ...        time_threshold=30)
+    >>> ms.load(["C14_flash_extent_density"])
+    >>> ms = ms.resample(ms.first_scene["C14"].attrs["area"])
+    >>> ms.save_animation("/path/for/output/{name:s}_{start_time:%Y%m%d_%H%M}.mp4")
+
+In this example, we pass to
+:meth:`~satpy.multiscene.MultiScene.from_files` the additional parameters
+``ensure_all_readers=True, group_keys=["start_time"], time_threshold=30``
+so we only get scenes at times that both ABI and GLM have a file starting
+within 30 seconds from each other, and ignore all other differences for
+the purposes of grouping the two.  For this example, the ABI files occur
+every 5 minutes but the GLM files (processed with glmtools) every minute.
+Scenes where there is a GLM file without an ABI file starting within at
+most ±30 seconds are skipped.
```

Comment:
Should this section mention the `BackgroundCompositor` that does the heavy lifting of "blending" the two instruments together?

Should you maybe also link to the `group_files` function again to point the user to the docstring for more information on these keyword arguments?

Reply:
Added several more lines, including with links to the `BackgroundCompositor` and `group_files`.